src/LiveApi.js

```diff
@@ -13,6 +13,8 @@ let WebSocket = typeof window !== 'undefined' ? window.WebSocket : MockWebSocket
 const shouldIgnoreError = error =>
     error.message.includes('You are already subscribed to') ||
         error.message.includes('Input validation failed: forget');
+const shouldNotThrow = json => 
+    json.error.code === 'SelfExclusion' && json.msg_type === 'authorize';
 
```

Comment:
@qingweibinary please read my previous comment. We are after the error code, type etc and this time around in the front/next-gen. Our error returns a JS Error object and code is just not part it. Just message if i could remember. Oh and the data above, i simply means response 😉